src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars-block stars--0">
```

Comment:
BEM modifier should only be used on element which has the corresponding block class
Checklist